cuda_bindings/setup.py

```diff
@@ -32,7 +33,13 @@
     raise RuntimeError("Environment variable CUDA_HOME or CUDA_PATH is not set")
 
 CUDA_HOME = CUDA_HOME.split(os.pathsep)
-nthreads = int(os.environ.get("PARALLEL_LEVEL", "0") or "0")
+if os.environ.get("PARALLEL_LEVEL") is not None:
+    warn(
+        "Environment variable PARALLEL_LEVEL is deprecated. Use CUDA_PYTHON_PARALLEL_LEVEL instead",
+        DeprecationWarning,
+        stacklevel=1,
+    )
+nthreads = int(os.environ.get("CUDA_PYTHON_PARALLEL_LEVEL", "0") or "0")
```

Comment:
This treatment ignored `PARALLEL_LEVEL`, so it means `PARALLEL_LEVEL` is not deprecated but removed.